benchmarks/playing_cards/procedural_c#/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+namespace procedural_c_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            PlayingCards p = new PlayingCards();
+        }
+    }
+
+    public class PlayingCards
+    {
+        string[] suits = { "Diamonds", "Spades", "Hearts", "Clubs" };
+        string[] values = { "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine", "Ten", "Jack", "Queen", "King", "Ace" };
+        string[] deck = new string[52];
+        int deckCount = 0;
+
+        public PlayingCards()
+        {
+            int count = 0;
+            for (int i = 0; i < 1000; i++)
+            {
+                createDeck();
+                while (deckCount > 0)
+                {
+                    string deck = showDeck();
+                    shuffle();
+                    dealCard();
+                    // Count variable is to make sure ShowDeck() is not optimised away
+                    count += (showDeck()).Length;

```

Comment:
```suggestion
                    string deck = showDeck();
                    shuffle();
                    dealCard();
                    // Count variable is to make sure ShowDeck() is not optimised away
                    count += deck.Length;

```
Den laver show to gange i stedet for en